strawberry/utils/__init__.py

```diff
@@ -0,0 +1,3 @@
+import graphql
+
+IS_GQL_32 = "3.3" not in graphql.__version__
```

Comment:
Maybe better to try to parse this? I'm almost sure we have a dependency which can transform this into a tuple like `(3, 3, ...)`. Then this could be a comparison like `version < (3, 3)`, meaning it would not break even if 3.4/4.0 gets released